lib/auth/webauthn/register.go

```diff
@@ -228,20 +228,34 @@ func upsertOrGetWebID(ctx context.Context, user string, identity RegistrationIde
 	return wla.UserID, nil
 }
 
+// RegisterRequest represents fields needed to register a new webautn device.
+type RegistrationRequest struct {
```

Comment:
Maybe:

```suggestion
// RegisterResponse represents fields needed to register a new webautn device.
type RegisterResponse struct {
```

as this is the last part of the ceremony, ie a response to Begin?